### DISCUSSION TITLE
How do I quickly return to the main repository screen when I haven't retrieved anything in the repository?

### DISCUSSION THREAD

**Discussion by SpiritLing** [community/community#62055] [Repositories] [answered]
### Select Topic Area

Product Feedback

### Body

![image](https://github.com/community/community/assets/26963582/d05ced98-b17d-4b42-bb0f-2cc609ae6ad8)

How do I quickly return to the main repository screen when I haven't retrieved anything in the repository?

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @SpiritLing , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by ahmetpalavan** [comment]
There are two different ways to quickly return to the main repository screen on GitHub without taking any files or directories from the repository. You can choose which way is more convenient for you:

1. **Using the Browser Back Button:**
   By clicking your browser's back button, you can go back to the page you last visited (the main repository screen). This method may be the simplest and fastest way.

2. **Going to GitHub Home Page:**
   To quickly return to the GitHub homepage, you can type "github.com" in the address bar of your browser and press Enter. This will redirect you to the GitHub homepage.

Note: These methods can be used to return to GitHub after you have not received any files or directories from any repository. After retrieving a file or directory from a repository, the browser back button or navigating to the GitHub homepage may affect your past actions. Therefore, when navigating within a repository, you can also go back to the main repository screen using the "Home" link in the top left corner of the page.

**Reply by SpiritLing** [reply]
Perhaps not understanding what I'm doing, I first went to a repository's home page, or clicked on the README.md file
- https://github.com/DuendeSoftware/IdentityServer
- https://github.com/DuendeSoftware/IdentityServer/blob/main/README.md

At this point I clicked on search in the upper right corner, for example, and searched for the keyword multiple times. When I searched again, he didn't return any results and I couldn't click on any file

https://github.com/search?q=repo%3ADuendeSoftware%2FIdentityServer+dwaddd&type=code

At this point I want to click on the folder to manually go to the jump file, there is no file on the right side, I can not click on the jump, I want to return to the following two addresses at this time
- https://github.com/DuendeSoftware/IdentityServer
- https://github.com/DuendeSoftware/IdentityServer/blob/main/README.md

There is no button on the interface to go back to the repository directly, and back requires multiple clicks, which doesn't meet my requirements. As for returning directly to the github.com homepage, it's even worse.

So, what I would like is to be able to go back to the repository's main interface or the repository's README.md when there is a single repository in the filter, so that I can quickly find it again manually in the repository.

> *Translated via Deepl, sorry if anything is inappropriate!*

**Comment by nirav-gajera** [comment]
* **Click the Back button in your browser.** This will take you back to the previous page you were on, which should be the main repository screen.
* **Press the **`Esc`** key on your keyboard.** This will also take you back to the previous page.
* **Click the **`Repository`** tab at the top of the page.** This will always take you to the main repository screen, regardless of what page you are currently on.

If you have retrieved something from the repository, you can still return to the main repository screen by clicking the **`Repository`** tab. However, you will first need to click the **`X`** button in the top right corner of the panel where the retrieved item is displayed.

Here are some additional tips for quickly returning to the main repository screen:

* If you are using GitHub Desktop, you can also use the **`Ctrl`+`Shift`+`R`** keyboard shortcut to return to the main repository screen.
* If you have a lot of repositories open, you can use the **`Repository`** dropdown menu to quickly select the repository you want to view.
* You can also use the search bar at the top of the page to search for a specific repository.

**Comment by SpiritLing** [comment, marked as answer]
I've figured out how to go directly back to the main repository screen by clicking on the red box in the image below

![image](https://github.com/community/community/assets/26963582/316f379e-859f-48dd-98f7-66c165fc6f90)
